internal/gmtcli/channel.go

```diff
@@ -89,13 +89,19 @@ func printChannels(channels []*gomeshproto.Channel) error {
 			fmt.Printf("%-15s| ", "N/A")
 		}
 		if channelInfo.Settings.ModuleSettings.GetPositionPrecision() > 0 {
-			fmt.Printf("%-15d| ", channelInfo.Settings.ModuleSettings.PositionPrecision)
+			fmt.Printf(
```

Comment:
Is this a Go formatting suggestion? I have `gotfmt` to format when save and it was fine with the single line. If there isn't a formatting precedent from Go to break up the lines like this can we keep it how it was?